hadoop-tools/hadoop-aliyun/src/main/java/org/apache/hadoop/fs/aliyun/oss/AliyunOSSFileSystemStore.java

```diff
@@ -147,6 +147,13 @@ public void initialize(URI uri, Configuration conf, String user,
       throw new IllegalArgumentException(msg);
     }
 
+    boolean redirectEnable = conf.getBoolean(REDIRECT_ENABLE_KEY,
+    REDIRECT_ENABLE_DEFAULT);
+    if (!redirectEnable) {
+      clientConf.setRedirectEnable(false);
+      LOG.info("oss redirectEnable closed");
```

Comment:
log at debug?

Reply:
I will change the log level to debug in the next commit